### PR TITLE
Add e2e tests for directory redirection

### PR DESCRIPTION
**Pull request by saw-jan**

### CODE REVIEW

File: tests/acceptance/features/webUICreateFilesFolders/createMdFile.feature [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    And the user has created folder "parent-folder"
+
+  Scenario: create a new md file
+    When the user reloads the current page of the webUI
+    When the user opens folder "parent-folder" using the webUI
+    Then there should be no resources listed on the webUI
+    When the user creates a md file with the name "sample.md" using the webUI
+    And the user closes the text editor using the webUI
+    # Then breadcrumb for folder "parent-folder" should be displayed on the webUI
+    Then the user should be in the root directory on the webUI
```
Comment (Talank) [inline tests/acceptance/features/webUICreateFilesFolders/createMdFile.feature:19]:
```suggestion
    Then the user should be in the root directory on the webUI
    # Then breadcrumb for folder "parent-folder" should be displayed on the webUI
```

File: tests/acceptance/pageObjects/filesPage.js [hunk 2]
```diff
@@ -315,6 +316,48 @@ module.exports = {
...
+      return this
+    },
+    closeTextEditor: function() {
+      return this.waitForElementVisible('@editorCloseBtn').click('@editorCloseBtn')
+    },
+    isRootDirectory: async function() {
+      let status
+      await this.isVisible('@breadcrumb', result => {
+        if (result.value === true) {
+          status = !result.value
```
Comment (Talank) [inline tests/acceptance/pageObjects/filesPage.js:355]: what about `status = false`? That will be less complex I think

File: tests/acceptance/pageObjects/filesPage.js [hunk 3]
```diff
@@ -315,6 +316,48 @@ module.exports = {
...
+        } else {
+          status = true
+        }
+      })
+      return status
```
Comment (kiranparajuli589) [inline tests/acceptance/pageObjects/filesPage.js:360]:
```suggestion
      let status = false
      await this.isVisible('@breadcrumb', result => {
          status = result.value === true
      })
      return status
```
this should do it!

File: tests/acceptance/stepDefinitions/filesContext.js [hunk 4]
```diff
@@ -1141,3 +1141,16 @@ When(
...
+  return client.page.filesPage().createMdFile(fileName)
+})
+
+When('the user closes the text editor using the webUI', function() {
+  return client.page.filesPage().closeTextEditor()
+})
+
+Then('the user should be in the root directory on the webUI', async function() {
+  const isRootDirectory = await client.page.filesPage().isRootDirectory()
+  assert.ok(isRootDirectory, 'Not in the root directory')
```
Comment (kiranparajuli589) [inline tests/acceptance/stepDefinitions/filesContext.js:1155]:
```suggestion
  const isUserInRootDirectory = await client.page.filesPage().isRootDirectory()
  assert.ok(isUserInRootDirectory, 'Expected user in the root directory but found elsewhere')
```

**Pull request closed without merging**